pkg/client/http/mapper.go

```diff
@@ -207,3 +207,19 @@ func setMapWithPath(targetMap map[string]interface{}, path string, val interface
 	}
 	return targetMap
 }
+
+func mapPrepare(mp config.MappingParam, rawTarget interface{}) (*requestParams, []string, string, []string, error) {
+	target, err := validateTarget(rawTarget)
+	if err != nil {
+		return nil, nil, "", nil, err
+	}
+	_, fromKey, err := client.ParseMapSource(mp.Name)
+	if err != nil {
+		return nil, nil, "", nil, err
+	}
+	to, toKey, err := client.ParseMapSource(mp.MapTo)
+	if err != nil {
+		return nil, nil, "", nil, err
+	}
+	return target, fromKey, to, toKey, nil
+}
```

Comment:
if returning params are too much, `
func funcname()(param1 string,param2 int){
    return
}
` may be clean and better

Reply:
and more comments?